uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/crm/app.form.workpattern.js

```diff
@@ -473,6 +559,53 @@ define([
       }
     };
 
+    /**
+     * Adjusts the time value for the specified type.
+     * It allows to select only times in the specific range:
+     * - "from" time 00:00 to 23:30
+     * - "to" time 00:15 to 23:45
+     *
+     * @param {String} timeType From|To
+     */
+    Day.prototype._adjustTimeValue = function (timeType) {
```

Comment:
same here about "adjust"

Reply:
I think this one is OK, the description of the function explain how it works.